lib/streamlit/runtime/state/query_params.py

```diff
@@ -132,6 +145,16 @@ def set_with_no_forward_msg(self, key: str, val: Union[List[str], str]) -> None:
         else:
             self._query_params[key] = str(val)
 
+    def check_both_apis_used(self):
```

Comment:
Same comment about the name being awkward as in the other similarly named method

This method should also be underscore-prefixed